.pipelines/cni/cilium/cilium-overlay-load-test-template.yaml

```diff
@@ -169,22 +200,48 @@ stages:
                   envsubst '${CILIUM_IMAGE_REGISTRY},${CILIUM_VERSION_TAG}' < test/integration/manifests/cilium/v${DIR}/cilium-operator/templates/deployment.yaml | kubectl apply -f -
                   kubectl get po -owide -A
 
-                  if [ -z "$TEST_AZURE_IPAM_VERSION" ]
-                  then
-                    echo "TEST_AZURE_IPAM_VERSION is not set, using default value"
-                    IPAM=$(make azure-ipam-version)
-                  else
-                    IPAM=$(TEST_AZURE_IPAM_VERSION)
-                    echo "TEST_AZURE_IPAM_VERSION is set to $IPAM"
-                  fi
-                  if [ -z "$TEST_CNS_VERSION" ]
+                  if [ ${{parameters.upgradeScenario}} = "true" ]
                   then
-                    echo "TEST_CNS_VERSION is not set, using default value"
-                    CNS=$(make cns-version)
+                    echo "Upgrade scenario is true, using upgrade azure ipam and cns version from pipeline variables"

```

Comment:
is this basically a copy of the above for the non-hubble case? Is there a way we can extract this logic to a template or similar? If we make a modification to one it could get desynced with the other case?